### DISCUSSION TITLE
School Wifi is fucking with dextensify

### DISCUSSION THREAD

**Discussion by Caelan51** [3kh0/ext-remover#1212] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE]
and before you start calling me a skid yes I followed the instructions and I am not a dumbass. In fact I have been using dextensify for a year but something is wrong with it for me. Apparently when I'm at school the exploit doesn't seem to work right its like even when I spam the file url switch it doesn't give me the desired effect. So i'm beginning to think that school Wifi now has the ability to fuck with Dextensify.  
And I have articles to make on UnAnything so I basically have 3 options which are 
A- Find a new exploit B- use a shitty proxy browser that isn't blocked by light speed but crashes on itself or C- wait until I go home to run the exploit
TLDR: School Wifi makes dextensify not work I need alternatives or a reason why the wifi is fucking with the exploit

**Comment by schoolexploitkid** [comment]
use ExtHang3r

**Reply by Caelan51** [reply]
ok I'll try this exthang3r thing

**Reply by Caelan51** [reply]
well that shit didn't fucking work I guess I was correct the wifi is fucking with hanging exploits

**Comment by S-PScripts** [comment]
dextensify is ass use something else
